src/main/target/common_pre.h

```diff
@@ -497,44 +493,38 @@ extern uint8_t _dmaram_end__;
 #define USE_RX_LINK_UPLINK_POWER
 
 #define USE_AIRMODE_LPF
-#define USE_CANVAS
-#define USE_FRSKYOSD
 #define USE_GYRO_DLPF_EXPERIMENTAL
 #define USE_MULTI_GYRO
 #define USE_SENSOR_NAMES
 #define USE_UNCOMMON_MIXERS
 #define USE_SIGNATURE
 #define USE_ABSOLUTE_CONTROL
 #define USE_HOTT_TEXTMODE
-#define USE_LED_STRIP_STATUS_MODE
-#define USE_VARIO
```

Comment:
What about this one? USE_VARIO is removed but doesn't have gating to activate it.

Reply:
It's now a custom define option